internal/storage/mysql_provider.go

```diff
@@ -25,12 +25,15 @@ func NewMySQLProvider(configuration schema.MySQLStorageConfiguration) *MySQLProv
 			sqlGetPreferencesByUsername:     fmt.Sprintf("SELECT second_factor_method FROM %s WHERE username=?", userPreferencesTableName),
 			sqlUpsertSecondFactorPreference: fmt.Sprintf("REPLACE INTO %s (username, second_factor_method) VALUES (?, ?)", userPreferencesTableName),
 
+			sqlUpgradesV002TOTPAlgorithm:       fmt.Sprintf("ALTER TABLE %s ADD algorithm VARCHAR(10) DEFAULT 'sha512' NOT NULL", totpSecretsTableName),
```

Comment:
Given issues with supported authenticator apps should we be considering a different default here?